contrib/images/e2e-test/run.sh

```diff
@@ -94,55 +94,12 @@ function e2e::wait-for-libvirt-domain {
   virsh -c qemu+tcp://virtlet/system list
 }
 
-function e2e::chat-with-vm {
-  expect -c '
-    set timeout 600
-    spawn virsh -c qemu+tcp://virtlet/system console cirros
-    expect {
-      timeout { puts "initial message timeout"; exit 1 }
-      "Escape character"
-    }
-    send "\r"
-
-    expect {
-      timeout { puts "login prompt timeout"; exit 1 }
-      "login:"
-    }
-    send "cirros\r"
-
-    set timeout 20
-    expect {
-      timeout { puts "password prompt timeout"; exit 1 }
-      "Password: "
-    }
-    sleep 3
-    send "cubswin:)\r"
-
-    expect {
-      timeout { puts "shell prompt timeout"; exit 1 }
-      -re "\n\\$"
```

Comment:
No, it failed before that. There's no single quotes in this file at all. This is just regexp for a dollar prompt. Double slash is here because of tcl string syntax which I'm not very good at, but I debugged this before when writing expect scripts for devbox testing